ElectricBillApi/Services/MyExtensions.cs

```diff
@@ -35,7 +19,8 @@ public static ElectricityProvider StringToProvider(this string name)
                     return provider;
                 }
             }
-            throw new NoProviderFoundException("No such electricityprovider exists.");
```

Comment:
Throwing was good. You just don't need to handle it in a controller, let it fail. It will fail with 500 naturally.

Reply:
Ok. Hopefully I understood it correctly now? However I dont get the 500 in Postman because VS will stop at the exception but it will be 500 when running compiled/standalone?